src/cms/cms.js

```diff
@@ -22,7 +22,7 @@ CMS.registerPreviewTemplate('contribute-your-ideas', CollectionsPagePreview)
 const loadFonts = () => {
   const primaryFontscript = document.createElement('link')
   primaryFontscript.href =
-    '//fonts.googleapis.com/css?family=Libre+Franklin:400,600'
+    '//fonts.googleapis.com/css?family=Libre+Franklin:400,600, 800'
```

Comment:
If we only needed for the 404 I will rather not pull it.
Besides, I think this only affects the CMS